pkg/config/config.go

```diff
@@ -1540,6 +1542,13 @@ func setupFipsEndpoints(config Config) error {
 
 	// HTTP for now, will soon be updated to HTTPS
 	protocol := "http://"
+	if config.GetBool("fips.https") {
+		protocol = "https://"
+		// top-level setting takes precedence
+		if config.GetBool("skip_ssl_validation") {
+			config.Set("skip_ssl_validation", !config.GetBool("fips.tls_verify"))
```

Comment:
We should overwrite it in any cases. Setting `fips.tls_verify` to false and `skip_ssl_validation` to true should skip the SSL check.

Reply:
I pushed a change for this; we decided to keep `fips.tls_verify` and we also decided that it should _always_ override `skip_ssl_validation`. The latest code reflects that behavior.